pkg/cri/resource-manager/policy/builtin/memtier/pod-preferences.go

```diff
@@ -40,6 +40,20 @@ const (
 	keyColdStartPreference = "cold-start"
 )
 
+// cpuClass is a type of CPU to allocate
+type cpuClass int
```

Comment:
nitpick: Could we get a `func (t cpuClass) String() string` added in there ? Would simplify printing `cpuClass`es.

Reply:
Good point!

There you go. Fixed `cpuClassNames[...]` from the patch to use the stringer.